plugins/rt.js

```diff
@@ -1572,6 +1576,10 @@
 				if (edata.responseBodyNotUsed) {
 					BOOMR.addVar("fetch.bnu", 1, true);
 				}
+
+				if (edata.responseUrl) {
+					BOOMR.addVar("ru", edata.responseUrl, true);
```

Comment:
I'd suggest adding a namespace before the `ru`, something like `xhr.ru` to avoid clashes with other plugins.